test/spec/base/base.js

```diff
@@ -12,5 +12,5 @@ function validateDiagram(results, done) {
   }
 }
 
-
-describeSuite('base', __dirname + '/diagrams', validateDiagram, { timeout: 4000 });
\ No newline at end of file
+// creating subprocess planes for complex diagram takes a while
```

Comment:
This might be hard to understand in the future. Shall we maybe summarize your findings in an issue and link from here. So something like:

* With xyz we introduced planes for subprocesses
* These increased import times in local testing by factor ~1.3 (30% more time needed)
* However, on CI-tests the time difference is significantly higher
* => TODO: investigate why performance difference is so high on phantomJS / when ran in CI

Reply:
Makes sense! I'll create an issue for it